llm/server.go

```diff
@@ -224,6 +224,11 @@ func NewLlamaServer(gpus gpu.GpuInfoList, model string, ggml *GGML, adapters, pr
 			gpuCount = 0
 		}
 
+		if strings.HasPrefix(servers[i], "oneapi") {
+			os.Setenv("Path", os.Getenv("Path")+dir)
+			slog.Debug("append oneapi lib in Path env:", os.Getenv("Path"))
+		}
```

Comment:
This doesn't seem necessary.   [This block](https://github.com/ollama/ollama/blob/main/llm/server.go#L260-L279) should cover it, and if you need additional deps wired up, use GpuInfo.DependencyPath.

Reply:
yes, you are right, i remove it and ollama also works even if we didn't source the oneapi-related env script.